js/apps/admin-ui/src/user/UserForm.tsx

```diff
@@ -327,37 +334,13 @@ export const UserForm = ({
             )}
           </FormGroup>
         )}
-
-        <ActionGroup>
-          <FormSubmitButton
-            formState={formState}
-            data-testid={!user?.id ? "create-user" : "save-user"}
-            isDisabled={
-              !user?.id &&
-              !watchUsernameInput &&
-              realm.registrationEmailAsUsername === false
-            }
-            allowNonDirty
-            allowInvalid
-          >
-            {user?.id ? t("save") : t("create")}
```

Comment:
here the button will say save or create, this is not the case anymore when you change it to the `FixedButtonsGroup`

Reply:
Introduced optional props for naming the buttons in FixedButtonGroup with the latest commit.